blog(embertimes132): related files hopper (#504)
ember-learn/ember-blog: source/2020-01-27-the-ember-times-issue-132.md

---

## [Related Files Hopper VSCode Extension 🤸](https://twitter.com/suchita009/status/1218310260592496641)

Ever needed to keep finding related files in your Ember repo using [VSCode](https://code.visualstudio.com/)? [Suchita Doshi (@suchitadoshi1987)](https://github.com/suchitadoshi1987) created an extension to make that leap easier!

> The [Related Files Hopper extension](https://marketplace.visualstudio.com/items?itemName=suchitadoshi1987.file-hopper) helps developers navigate through the multiple files that are touched when working on a feature. Within Ember for example, a person adding or removing something in `components/foo.js` would likely also touch `templates/foo.hbs` and related tests `unit/foo-test.js` or `acceptance/foo-test.js`. This extension provides hot keys to present related files as an option to jump into while you’re currently working in a selected file.

Reviewer: Hmm, maybe suggest using a rendering test for testing a component, so that a reader doesn't need to think that the developer, for example, may have extracted methods into a utility for unit testing?

 In the last sentence, I used your component file to make your idea more concrete. I think my edits convey what you had meant, but just in case, can you check this? Thanks.

```suggestion
The [Related Files Hopper extension](https://marketplace.visualstudio.com/items?itemName=suchitadoshi1987.file-hopper) helps developers navigate through the multiple files that are touched when working on a feature. Within Ember for example, a person adding or removing something in `components/foo.js` would likely also touch `templates/foo.hbs` and related tests `integration/foo-test.js` or `acceptance/foo-test.js`. This extension provides hot keys for related files when you're working in `components/foo.js`.
```

Author: ah yeah good point copied the example on github :p
